src/registrar/tests/test_admin.py

```diff
@@ -1786,6 +1786,8 @@ def test_readonly_when_restricted_creator(self):
                 "updated_at",
                 "status",
                 "rejection_reason",
+                "updated_federal_agency",
```

Comment:
We'll keep both fields and make the new field visible only to superadmins

Reply:
Yes we are keeping both fields for now, but we are going to comment out #updated_federal_agency after we get an approval so it doesn't show up for anyone on Admin since it's not useful

Reply:
AC #2: superusers can see this but not analysts (this will just be empty for now so we don't need to show it to them and add confusion)

Reply:
OH yes sorry this AC was not updated - Erin and I had a meeting with @abroddrick and agreed with that compromise! Apologies for the confusion!

Reply:
Got it. APPROVED.